src/ResourceManager/Sql/Commands.Sql/Security/Model/BaseAuditingPolicyModel.cs

```diff
@@ -14,43 +14,27 @@
 
 namespace Microsoft.Azure.Commands.Sql.Security.Model
 {
-    public class AuditingPolicy
-    {
-       
-        public AuditingPolicy()
-        {
-            ConnectionStrings = new ConnectionStrings();
-        }
+    // types of storage keys
 
-        public string ResourceGroupName { get; set; }
+    public enum StorageKeyKind { Primary, Secondary };
```

Comment:
Please add proper Doc comments ( /// ) to all of these enums and their properties.  I realize it will add a lot of lines to the file, but it will significantly help people understand what everything means and does.